baremetal/remote/remote_test.go

```diff
@@ -72,6 +72,7 @@ preferences: {}
 users:
 - name: kubernetes-admin
 `
+			_ = validKubeConfig
```

Comment:
This looks a bit weird to me. Is the linter complaining? I can see `calidKubeConfig` being used below so should be fine. Maybe the linter didn't get it?

Reply:
@furkatgofurov7 do you remember why did I do this, I think you were there when we did this but I have forgotten the reason unfortunately.

Reply:
Yes, there was a problem with one of the linters, complaining "declared but not used" kind of error ?

Reply:
@kashifest do you still see that error locally without this change? I just checked it in your branch and have not seen this. I even added removed linters back from the golangci.yaml and removed this change thinking one of those were throwing it, but everything went fine. Can you verify it locally as well?